packages/ui/src/utils/connectors/social-connectors.ts

```diff
@@ -0,0 +1,35 @@
+import { SearchParameters } from '@/types';
```

Comment:
📌 This code existed already, split the files down into base, social and blockchain